installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with network customization options. The installation program allows for some customization to be applied at the installation stage.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[Installing a cluster on vSphere in a restricted network]**: You can install a cluster on VMware vSphere infrastructure in a restricted network by creating an internal mirror of the installation release content.
+ You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
+=== User-provisioned infrastructure installation of {product-title} on vSphere
+
+* **xref:../../installing/installing_vsphere/installing-vsphere.adoc#[Installing a cluster on vSphere]**: You can install {product-title} on VMware vSphere infrastructure that you provision.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installing-vsphere-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on VMware vSphere infrastructure that you provision with customized network configuration options.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[Installing a cluster on vSphere in a restricted network]**: {product-title} can be installed on VMware vSphere infrastructure that you provision in a restricted network.
+
+=== Uninstalling an installer-provisioned infrastructure installation of {product-title} on vSphere
+
+* **xref:../../installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc#uninstalling-cluster-vsphere-installer-provisioned[Uninstalling a cluster on vSphere that uses installer-provisioned infrastructure]**: You can remove a cluster that you deployed on VMware vSphere infrastructure that used installer-provisioned infrastructure.
+
+[id="preparing-to-install-on-vsphere-next-steps"]
+== Next steps
+
+* xref:../../installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc#vsphere-problem-detector-viewing-events_vsphere-problem-detector[Use the vSphere Problem Detector Operator] to determine if the cluster has permission or storage configuration issues.
```

Comment:
My view is that this next steps section is a list of steps to take after reading this assembly, rather than after having completed the installations recommended in the links that are referenced on this page. You could change these next steps to list the next step in the reader's journey after having read this assembly. Or, you could remove the next steps section and the reader will instead move onto the respective installation assembly that is suitable for their business needs.

Reply:
Removed. Not just because I agree with your perspective on the meaning of the page (it's not intended to be the literal reader journey), but because it got to be very large and replicates the information at the destination. "Preparing to install" does not equal "Next steps".